node/index.js

```diff
@@ -96,250 +100,252 @@ app.post('/api/create_link_token', function (request, response, next) {
   if (PLAID_ANDROID_PACKAGE_NAME !== '') {
     configs.android_package_name = PLAID_ANDROID_PACKAGE_NAME;
   }
-
-  client.createLinkToken(configs, function (error, createTokenResponse) {
-    if (error != null) {
-      prettyPrintResponse(error);
-      return response.json({
-        error: error,
-      });
-    }
-    response.json(createTokenResponse);
-  });
+  try {
+    const createTokenResponse = await client.linkTokenCreate(configs);
+    response.json(createTokenResponse.data);
+  } catch (error) {
+    prettyPrintResponse(error);
+    return response.json({
+      error: error.response.data,
+    });
+  }
 });
 
 // Create a link token with configs which we can then use to initialize Plaid Link client-side.
 // See https://plaid.com/docs/#payment-initiation-create-link-token-request
 app.post(
   '/api/create_link_token_for_payment',
-  function (request, response, next) {
-    client.createPaymentRecipient(
-      'Harry Potter',
-      'GB33BUKB20201555555555',
-      {
-        street: ['4 Privet Drive'],
-        city: 'Little Whinging',
-        postal_code: '11111',
-        country: 'GB',
-      },
-      function (error, createRecipientResponse) {
-        const recipientId = createRecipientResponse.recipient_id;
+  async function (request, response, next) {
+    try {
+      const createRecipientResponse = await client.paymentInitiationRecipientCreate(
+        {
```

Comment:
there's a mix of inline request objects v.s. defining the objects on their own line (`config = {...` pattern). whichever pattern you prefer, should stick with it! 

i personally prefer inline as it's more concise, but the separate line has a clear benefit for being more debuggable.

Reply:
done. thanks!